.github/workflows/pr-prefix.yml

```diff
@@ -10,18 +10,21 @@ jobs:
     steps:
       - name: Check PR title prefix
         run: |
-          if echo "${{ github.event.pull_request.title }}" | tr ':' '\n' | head -n 1 | grep -qE 'feat|fix|docs|refactor|ci|chore(\([^)]+\)|$)';
+          if echo "${{ github.event.pull_request.title }}" | tr ':' '\n' | head -n 1 | grep -qE '^(feat|fix|docs?|refactor|ci|chore(\([^)]+\))?|deps)$';
```

Comment:
In fact, the chore: form was already possible.
https://github.com/lablup/backend.ai/actions/runs/9676696895/job/26696837465?pr=2310
But I think your way is a better way!

Reply:
In my case, it didn't work:
```console
$ echo 'chore: asdf' | tr ':' '\n' | head -n 1 | grep -E '^(feat|fix|docs?|refactor|ci|chore(\([^)]+\))?|deps)$'
chore

$ echo 'chore: asdf' | tr ':' '\n' | head -n 1 | grep -qE 'feat|fix|docs|refactor|ci|chore(\([^)]+\)|$)'
```